optaplanner-integration/optaplanner-workbench-models/optaplanner-workbench-models-core/pom.xml

```diff
@@ -10,7 +10,7 @@
     <version>7.31.0-SNAPSHOT</version>
   </parent>
 
-  <!-- TODO Remove and replace by optaplanner-core-gwt once the workbench supports net.ltgt.gwt.maven:gwt-maven-plugin -->
+  <!-- TODO Remove and replace by ? once the workbench supports net.ltgt.gwt.maven:gwt-maven-plugin -->
```

Comment:
We'll never go to net.ltgt.gwt.maven:gwt-maven-plugin, so this will never happen.
Let's remove this comment.

Reply:
Removed